### ISSUE TITLE
Saving and reading config to and from the generated readmes

### ISSUE THREAD

**Issue by rishavanand** [rishavanand/github-profilinator#52]
**Is your feature request related to a problem? Please describe.**
When users want to modify their readme already created with GitHub Profilinator, they need to start from scratch as there is no config option. 

**Describe the solution you'd like**
At markdown generation, the config can be added to it as comments then it can be loaded directly by providing the link to the existing readmes generated via GitHub Profilinator.

**Describe alternatives you've considered**
An alternative would be allowing users to download the config but again they will have to manage storage, add it to the readme as comment looks like a better solution. 

**Additional context**
none

**Comment by smkamranqadri** [comment]
Agreed with not going with config file down solution and about the readme, user have to share the raw file link in order us to get text instead of html and find the config area and parse it.

Another solution can use adding login with github and then save the config using backend may be firebase if not our own.

**Comment by rishavanand** [comment]
@smkamranqadri That's another good solution but I feel it increases the complexity of the solution by involving a backend. The config as a comment solution is simple to implement and manage. What do you think? Do you see any drawbacks to this solution?

**Comment by smkamranqadri** [comment]
A part from user to give us raw file link, nothing I can think of.

**Comment by rishavanand** [comment]
I've added TODOs for this issue. See if you can take them up.
